packages/client/src/pages/SearchResult/SearchResult.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import './SearchResult.scss';
+import TrecipeCard from '../../components/TrecipeCard/TrecipeCard';
+import { RootState } from '../../redux';
+import { connect } from 'react-redux';
+import { bindActionCreators, Dispatch } from 'redux';
+import { Link, withRouter, RouteComponentProps } from 'react-router-dom';
+import { fetchResultRequest } from '../../redux/SearchResult/action';
+import Trecipe from '../../../../shared/models/trecipe';
+import Destination from '../../../../shared/models/destination';
+import { DestinationCard } from '../Map/DestinationCard/DestinationCard';
+
+type SearchResultProps = ReturnType<typeof mapStateToProps> &
+    ReturnType<typeof mapDispatchToProps> &
+    RouteComponentProps<{ searchKey: string }>;
+class SearchResult extends React.Component<SearchResultProps, {}> {
+    componentDidMount(): void {
+        let searchKey = this.props.location.search;
+        console.log('1');
+        console.log(searchKey);
+        searchKey = searchKey.substring(3);
+        console.log('2');
+        console.log(searchKey);
+        this.props.fetchResultRequest(searchKey);
+    }
+    render() {
+        console.log('here');
+        return (
+            <div className="search-result-wrapper">
+                <div className="content">
+                    <div className="result-wrapper">
+                        <h1 className="page-title">Trecipes</h1>
+                        <div className="cards-wrapper">
+                            <ul className="card-list">
+                                {this.props.results?.trecipeResult.map((trecipe: Trecipe) => (
+                                    <li className="card-item" key={trecipe.uuid}>
+                                        <Link className="router-link" to={trecipe.uuid}>
+                                            <TrecipeCard {...trecipe} isReadOnly={true} />
```

Comment:
What if the trecipe is owned by the currently logged in user? Two options;
1. Make the TCs all readonly for result page, but have Trecipe Page that it links to editable
2. Make both TC and Trecipe Page editable if logged-in user owns this trecipe
@knox153 @shumalll What do you guys think i guess? I'm slightly leaning toward 1.

Reply:
1111, I prefer having all the TC look the same 🙂